comptests/prov/test_prov_k8s_cortx_cold_upgrade.py

```diff
@@ -184,10 +185,10 @@ def test_37352(self):
                     "Step 6: Start upgrade and verify pod went into crashloopbackoffstate or not")
                 LOGGER.info("Upgrading CORTX image to version: %s.",
                             self.cortx_all_image)
-                resp = self.deploy_lc_obj.upgrade_software(self.master_node_obj,
-                                                           self.prov_deploy_cfg["git_remote_path"],
-                                                           upgrade_type="cold")
+                resp = self.upgrade_lc_obj.upgrade_software(self.master_node_obj,
```

Comment:
We should preferably avoid `lc` word in the variable names. If its a lot of refactoring and needed across all the code then we might want to differ it. But if its in a small context, let's do it.

Reply:
done